### PR TITLE
added context menu support (right click on user -> apps)

### PR DESCRIPTION
**Pull request by Talhoid**
gthis pr adds contextmenu suppport

### CODE REVIEW

File: src/main/java/org/meteordev/meteorbot/command/Commands.java [hunk 1]
```diff
@@ -43,7 +45,14 @@ public void onReady(@NotNull ReadyEvent event) {
             commandData.add(command.build(new CommandDataImpl(command.name, command.description)));
         }
 
-        MeteorBot.BOT.updateCommands().addCommands(commandData).complete();
+        MeteorBot.BOT.updateCommands()
+            .addCommands(commandData)
+            .addCommands(
+                Commands.context(Command.Type.USER, "Tell them to read FAQ"),
+                Commands.context(Command.Type.USER, "Send installation guide"),
+                Commands.context(Command.Type.USER, "No old versions")
```
Comment (SIMULATAN) [inline src/main/java/org/meteordev/meteorbot/command/Commands.java:53]: I'd make the names constants and optionally move them to the respective command implementations for more maintainability

Comment (Talhoid) [reply src/main/java/org/meteordev/meteorbot/command/Commands.java:53]: hm, ok

Comment (Talhoid) [reply src/main/java/org/meteordev/meteorbot/command/Commands.java:53]: wait, but whrere would i move them

Comment (Talhoid) [reply src/main/java/org/meteordev/meteorbot/command/Commands.java:53]: ill just make them constants

**Comment by Talhoid** [comment]
made them constants

**Pull request closed without merging**